### ISSUE TITLE
What is your exp/hr

### ISSUE THREAD

**Issue by MrMegaMango** [finndev/PokeBuddy#1174]
Mine is 2000+ if its working normally. When that number fells to around 1k, something is wrong, either bot crashed i need to restart it, or I'm getting banned

**Comment by MariaBitterlich** [comment]
Running currently with 14k+

**Comment by Rab89** [comment]
currently 29k @ santa monica

**Comment by karimavine** [comment]
@Rab89 @MariaBitterlich at what speed guys >?

**Comment by Rab89** [comment]
im using 20 like i have always used

**Comment by vaizard12** [comment]
33k @ tokyo with speed of 15kmh.

**Comment by MariaBitterlich** [comment]
@vaizard12, be careful, because in another issue a user calculated that km/h is wrong and Pokebuddy runs with your 15 as 15 m/s --> 54 km/h In that way the speed warning is triggered and that seems to cause ban problems...

**Comment by vaizard12** [comment]
Yep, I know that, but I never get banned, soft nor perma. :) Most probably becuase I'm botting for 30mins to 1hr then 30mins break then bot again.
